### ISSUE TITLE
security issue 

### ISSUE THREAD

**Issue by Rohazon** [koying/SPMC#908]
Hello all,
I heard kodi has launch an update for some security issue, and was wondering if spmc was vulnerable too
https://forums.geforce.com/default/topic/1010134/shield-tv/kodi-popcorntime-vlc-stremio-subtitles-hack-exploit/

**Comment by ryadre** [comment]
Yes I would also like to know, also is it an issue if we dont use subtitles at all? Dont really want to leave spmc if I dont need to.

**Comment by Skyrocket2Paradise** [comment]
I hope so as well an issue report has already been created here too: [https://github.com/koying/SPMC/issues/905](https://github.com/koying/SPMC/issues/905)

**Comment by Rohazon** [comment]
sorry for the double post then

**Comment by sammarbella** [comment]
If you use Android you should be safe I think the subtitles exploit only affect Windows OS builds.

**Comment by Skyrocket2Paradise** [comment]
the exploit exists in all builds not just windows. [You can see the kodi team has pushed the security fixes to their android app as well](https://play.google.com/store/apps/details?id=org.xbmc.kodi&hl=en)

**Comment by sammarbella** [comment]
You are right, the exploit is present in all OS builds but his negative effect is only active in Windows OS.

Let me rephrase it:

"If you use Android you should be safe I think the subtitles exploit **has only effect in Windows OS**.

**Comment by unlocKing** [comment]
You think??? 

Please state facts and do not speculate.

**Comment by sammarbella** [comment]
Feel free to prove my thoughts are wrong with facts...i.e Android OS devices hacked using this exploit running Kodi/SPMC or any of his "flavors"

Until then our post is like mine: simply your opinion not backed by facts.

**Comment by unlocKing** [comment]
Relax dude!

I think you need to read carefully what you wrote and **how you wrote it**.

**Comment by sammarbella** [comment]
Thanks to care about my peace of mind.

I can't be more relaxed, I was about to fall asleep after a night shift.

Good night......Zzzzzz

**Comment by NedScott** [comment]
_All_ platforms are affected by the bug. The proof of concept was demonstrated on Windows, but is not limited to Windows.

**Comment by TinderboxUK** [comment]
Great, I watch a lot of Anime and most of it has English subtitles so 16.6.1 cannot come sooner.

What about my older android devices, I have a couple of TV boxes on kitkat 4.4 and they will not run 16.6.1 when it is released.

John.
